data/INRA/2016_harvest_data.ttl

```diff
@@ -0,0 +1,53 @@
+ <data/winemaking/PechRouge/Harvest/2016> a qb:DataSet; 
+  rdfs:label "Data about Harvest done on Pech Rouge in 2016"; 
+  qb:structure <DSD/harvestWeight/2016> . 
+ 
+<DSD/harvestWeight/2016> a qb:DataStructureDefinition; 
+  qb:component [qb:dimension bdg:plot                                                              ]; 
+  qb:component [qb:dimension bdg:variety                                                           ]; 
+  qb:component [qb:dimension bdg:date                                                              ]; 
+  qb:component [qb:dimension bdg:humanSensor                                                       ]; 
+  qb:component [qb:measure   bdg:grapeClusterWeight    ; qb:componentAttachment qb:DataSet         ]; 
+  qb:component [qb:attribute sosa:hasFeatureOfInterest ; qb:componentAttachment qb:MeasureProperty ]; 
+  qb:component [qb:attribute sdmx-attribute:unitMeasure; qb:componentAttachment qb:MeasureProperty ] . 
+ 
+<variety/Chardonnay> a bdg:Variety; 
+rdfs:label"Chardonnay"^^xsd:string . 
+ 
+<variety/Muscat-Petit-Grain> a bdg:Variety; 
+rdfs:label"Muscat-Petit-Grain"^^xsd:string . 
+
+<INRA/agent/fabien.robert> a bdg:humanSensor, ofpe:Operator, foaf:Person;
+  foaf:familyName "robert"^^xsd:string;
+  foaf:firstName "fabien"^^xsd:string .
+
+<INRA/agent/jerome.cerutti> a bdg:humanSensor, ofpe:Operator, foaf:Person;
```

Comment:
Let's hope that there is ever only one jerome cerutti from inra working on the grapes :)

Reply:
We can create a generated specific id for each person who created an observation, in fact it will be better.

Reply:
It will not. from experience it is allways better to avoid using uuids. Fo one thing whenever you generate a uuid, you have to start keeping track of it to be able to attach more triples to the node.
Here I was sugesting more that you look at your data and estimate the likelihood of having conflicting names and maybe add something more to the uri, such as a person id  or something... but looking at the actuial data there is only a name. 
So just leave it for now and maybe we will have to come back ot this later

Reply:
However be careful as you have both missing values and something named "nans" in the data

Reply:
Okay I'll check the data.

Reply:
The commit ca20410e9030b146b64be8384ff06b0b4eafffdd correct this.